lib/ItemCard.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+void main() {
```

Comment:
Waarom `main()` in deze file?
En runApp? Je hoeft hier alleen maar een class te maken voor die itemcard.